### PR TITLE
*: udpate `begin` and `commit` statements

### PR DESCRIPTION
**Pull request by zimulala**
### What is changed, added or deleted? (Required)
Udpate `begin` and `commit` statements.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### CODE REVIEW

**Comment by TomShawn** [comment]
LGTM
@crazycs520 @zz-jason @kissmydb PTAL

**Comment by TomShawn** [comment]
@zimulala Please resolve the conflicts in this PR.

**Comment by zimulala** [comment]
@TomShawn 
Done.

File: sql-statements/sql-statement-commit.md [hunk 1]
```diff
@@ -61,8 +65,7 @@ Query OK, 0 rows affected (0.01 sec)
 
 ## MySQL 兼容性
 
-* 在 MySQL 中，除了有多个 primary 的群组复制以外，`COMMIT` 语句通常不会导致错误。相比之下，TiDB 使用乐观并发控制，冲突可能导致 `COMMIT` 返回错误。
-* 默认情况下，`UNIQUE` 和 `PRIMARY KEY` 约束检查将延迟直至语句提交。可通过设置 `tidb_constraint_check_in_place=TRUE` 来改变该行为。
+* 对于 `COMMIT` 语句中的 `CompletionTypeWithinTransaction` 选项，目前只在语法上支持。
```
Comment (yikeke) [inline sql-statements/sql-statement-commit.md:68]: Need to be updated in https://github.com/pingcap/docs/pull/2922/ @zimulala

Comment (zimulala) [reply sql-statements/sql-statement-commit.md:68]: 在此 PR 之前有人已经加了：https://github.com/pingcap/docs/pull/2922/files#diff-4bc4de2792f4bfc92c0f73ec06393007R44

Comment (yikeke) [reply sql-statements/sql-statement-commit.md:68]: 那前两行得删去吧？

**Pull request merged**